### PR TITLE
Chore/update docs

### PR DESCRIPTION
**Pull request by BCerki**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

File: packages/button-theme/src/styles.tsx [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+
+      input: `
+          font-size: 0.8rem;
+        `,
+      header: `
+              font-size: 1rem;
+            `,
+      content: `
+              font-size: 0.8rem;
+              padding: 0.4rem;
```
Comment (tmastrom) [inline packages/button-theme/src/styles.tsx:233]: Added padding to content to ensure the close button does not extend beyond the container

File: packages/button-theme/src/styles.tsx [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+                `,
+      input: `
+                font-size: 1rem;
+              `,
+      header: `
+              font-size: 1.2rem;
+            `,
+      content: `
+              font-size: 1rem;
+              padding: 0.6rem;
```
Comment (tmastrom) [inline packages/button-theme/src/styles.tsx:258]: Added padding to content to ensure the close button does not extend beyond the container

File: packages/button-theme/src/styles.tsx [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+                `,
+      input: `
+              font-size: 1.2rem;
+            `,
+      header: `
+              font-size: 1.4rem;
+            `,
+      content: `
+              font-size: 1.2rem;
+              padding: 0.8rem;
```
Comment (tmastrom) [inline packages/button-theme/src/styles.tsx:283]: Added padding to content to ensure the close button does not extend beyond the container

**Review by tmastrom — DISMISSED** [review]

**Review by joshgamache — DISMISSED** [review]
Changes to the docs make sense. The extended and relocated Storybook section helps clarify how to actually get it going without digging into the scripts. 🌮 

I saw the discussion in Toolkit (Open) re: _Alert vs Notification_, I'm assuming the changes here to use the core notification component are just deduplication?

**Comment by BCerki** [comment]
> 

Yes, I'll fix the notification/alert imports

**Review by joshgamache — APPROVED** [review]
➕👍

**Pull request merged**